app/presenters/tree_builder_report_reports.rb

```diff
@@ -0,0 +1,64 @@
+class TreeBuilderReportReports < TreeBuilderReportReportsClass
+  private
+
+  def initialize(name, type, sandbox, reports_menu, group_title, build = true)
+    @rpt_menu  = reports_menu
+    @grp_title = group_title
+    super(name, type, sandbox, build = true)
```

Comment:
The `build = true` should be just `build` here, otherwise we're always passing true.